### PR TITLE
run tests pointing to filter members PR

### PR DESCRIPTION
**Pull request by melisabok**

### CODE REVIEW

**Comment by melisabok** [comment]
@carlosmondra tests are passing now!

File: .circleci/config.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ jobs:
             sudo apt-get install -y mongodb-org=4.4.5 mongodb-org-server=4.4.5 mongodb-org-shell=4.4.5 mongodb-org-mongos=4.4.5 mongodb-org-tools=4.4.5
             mongo mongodb://localhost:27017 --eval "rs.initiate()"
       - run: pip install ~/openreview-py-repo
-      - run: git clone https://$OPENREVIEW_GITHUB@github.com/openreview/openreview-api-v1.git ~/openreview
+      - run: git clone -b feature/filter-members https://$OPENREVIEW_GITHUB@github.com/openreview/openreview-api-v1.git ~/openreview
```
Comment (carlosmondra) [inline .circleci/config.yml:28]: We need to first merge https://github.com/openreview/openreview-api-v1/pull/2479
and then revert this change right?

**Review by carlosmondra — APPROVED** [review]
